pypeit/core/wavecal/wvutils.py

```diff
@@ -402,7 +402,8 @@ def zerolag_shift_stretch(theta, y1, y2):
     corr_denom = np.sqrt(np.sum(y1*y1)*np.sum(y2_corr*y2_corr))
     corr_norm = corr_zero/corr_denom
     if corr_denom == 0.0:
-        msgs.error('The shifted and stretched spectrum is zero everywhere. Cross-correlation cannot be performed. There is likely a bug somewhere')
+        msgs.warn('The shifted and stretched spectrum is zero everywhere. Cross-correlation cannot be performed. There is likely a bug somewhere')
+        raise RuntimeError()
```

Comment:
I didn't want to use msgs.error because it closes the log (at least until the setup GUI is merged in), and I just picked an exception without thinking of PypeItError. But you're right, that's a better exception so I'll raise that instead.